pkg/platform/yaml/yaml.go

```diff
@@ -32,7 +33,7 @@ type Yaml struct {
 	tele     *telemetry.Telemetry
 }
 
-func NewYamlStore(tcsPath string, mockPath string, tcsName string, mockName string, Logger *zap.Logger, tele *telemetry.Telemetry) platform.TestCaseDB {
+func NewYamlStore(tcsPath string, mockPath string, tcsName string, mockName string, Logger *zap.Logger, tele *telemetry.Telemetry) interface{} {
```

Comment:
Here can we have the return type as Yaml only and since it will implement the platform.TestCaseDb interface it can be passed as argument for the type param.